dataflow/gpu-workers/pytorch-minimal/README.md

```diff
@@ -0,0 +1,45 @@
+# PyTorch GPU minimal pipeline
+
+## Before you begin
+
+Make sure you have followed the
+[Dataflow setup instructions](../../README.md).
+
+Finally, save your resource names in environment variables.
+
+```sh
+export PROJECT=$(gcloud config get-value project)
+```
+
+## Building the Docker image
+
+We use Cloud Build to build the container image for the workers.
+
+```sh
+gcloud builds submit --config build.yaml
+```
+
+## Running the Dataflow job with GPUs
+
+We use Cloud Build to run the Dataflow job.
+We launch the job using the worker image to make sure the job launches
+with the same Python version as the workers.
+
+```sh
+export REGION="us-central1"
+export WORKER_ZONE="us-central1-f"
+export GPU_TYPE="nvidia-tesla-t4"
+
+gcloud beta builds submit \
+    --config run.yaml \
+    --substitutions _REGION=$REGION,_WORKER_ZONE=$WORKER_ZONE,_GPU_TYPE=$GPU_TYPE \
+    --no-source
+```
+
+> ℹ️ Make sure the GPU type you choose is available in the worker zone for the job.
```

Comment:
Accordingly, GPUs should be available in some zones in this region.

Reply:
Thanks, removed this note.